src/Dialogs/FileManager.cpp

```diff
@@ -191,7 +171,8 @@ class ManagedFileListWidget
   bool some_out_of_date;
 #endif
 
-  FileRepository repository;
+  RepoFileManager repomanager;
+  const FileRepository &repository = repomanager.repository;
```

Comment:
What's the point of this field?

Reply:
It prevents having `repomanager.repository` all over the place when iterating over the available files, which I think is a bit ugly.